### PR TITLE
Add freeipscanner

### PR DESCRIPTION
**Pull request by QU35T-code**
# Description

Addition of https://github.com/scrt/freeipscanner
Addition of https://github.com/scrt/scrtdnsdump in draft

### CODE REVIEW

**Review by ShutdownRepo — APPROVED** [review]

File: sources/install/package_ad.sh [hunk 1]
```diff
@@ -1244,4 +1263,6 @@ function package_ad() {
     install_netexec                # Crackmapexec repo
     install_extractbitlockerkeys   # Extract Bitlocker recovery keys from all the computers of the domain
     install_LDAPWordlistHarvester
+    install_freeipscanner
+    # install_scrtdnsdump          # This tool is a fork of adidnsdump (https://github.com/dirkjanm/adidnsdump). We are currently waiting to see if a PR will be made.
```
Comment (gbe) [inline sources/install/package_ad.sh:1267]: A PR was made then closed by adidnsdump' author: https://github.com/dirkjanm/adidnsdump/pull/15

Comment (ShutdownRepo) [reply sources/install/package_ad.sh:1267]: Not sure we can. Fork relies on scapy which is GPLv2, and we are GPLv3. I think we can’t…

Comment (QU35T-code) [reply sources/install/package_ad.sh:1267]: For now, can we merge freeipscanner and let scrtdnsdump as draft ?

Comment (ShutdownRepo) [reply sources/install/package_ad.sh:1267]: yup

**Pull request merged**